### ISSUE TITLE
Additional steps required

### ISSUE THREAD

**Issue by tmccoy00** [InitialState/pi-health-dashboard#1]
I followed this tutorial and had a number of issues along the way that required additional package installation that are not listed in the main tutorial steps - which will frustrate many users trying to get this going from scratch!

Part 1. Initial State Step 2: Install the ISStreamer

Following the detailed manual steps indicated I didn't have the correct version of pip and thus ISstreamer didn't install with the automatic script.

The key commands to get it working are

$ sudo apt-get install python-setuptools
$ sudo easy_install pip
$ sudo pip install ISStreamer

Part 2. psutil

While the pio initial script seemed to work, an error regarding python.h not found indicates the python development libraries are not install. Thus again, additional commands are needed from the more detailed steps.

https://github.com/giampaolo/psutil/blob/master/INSTALL.rst

$ sudo apt-get install gcc python-dev
$ pip install psutil

...to which the result of the tutorial instructions can then be followed.

**Comment by jrbail01** [comment]
What version of Raspbian (or other OS) did you run this on? Raspbian Jessie should have pip and the python development tools installed by default. 

I will add these additional steps as tips in the tutorial. Thank you for posting this!!

**Comment by tmccoy00** [comment]
pi@raspberrypi:~ $ uname -a
Linux raspberrypi 4.1.18-v7+ #846 SMP Thu Feb 25 14:22:53 GMT 2016 armv7l GNU/Linux
pi@raspberrypi:~ $ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 8 (jessie)"
NAME="Raspbian GNU/Linux"
VERSION_ID="8"
VERSION="8 (jessie)"
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

**Comment by jrbail01** [comment]
When you first ran the tutorial, did you run the script:
$ \curl -sSL https://get.initialstate.com/python -o - | sudo bash
?
I ask because that script should have handled installing python-setuptools, pip, easy_install, making sure pip is the right version, etc. Check the contents of https://get.initialstate.com/python to see where it should have installed everything you had to install manually.

**Comment by tmccoy00** [comment]
Yes, but it generated errors and didn't fully complete or install ISstreamer properly which I worked out when trying to run the streamer example.

**Comment by jmaybe** [comment]
Similar to above problems, but corrected with a slightly different solution.

Just set up a new pi 3 with Jessie and ran into psutil problem; it failed to properly install with pip. (I did not copy the error before correcting the problem...)
In order to make it install properly I ran the following:

sudo apt-get install build-essential python-dev python-pip

then

sudo pip install psutil

Corrected the problem and was up-and-running in no time.

R/
John
